install.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
I'm assuming this is the change from your other PR. It'll go away if we check the other one in, first.